res/huroutes.js

```diff
@@ -35,18 +35,39 @@ const huroutes = {
         },
         // A list of navigation service providers that can be chosen for the "navigate to" links'.
         // The default provider used is the first one.
+        // The getLink attribute is function(destination, waypoints).
+        // The waypoints attribute defines the number of waypoints passed to the getLink function.
         'navLinkProviders': {
-            'Google': coord => {
-                const link = 'https://www.google.com/maps/dir/?api=1&travelmode=driving&destination={0},{1}';
-                return link.format(coord.lat, coord.lng);
+            'Google': {
+                'getLink': dst => {
+                    const link = 'https://www.google.com/maps/dir/?api=1&travelmode=driving&destination={0},{1}';
+                    return link.format(dst.lat, dst.lng);
+                },
+                'waypoints': 0
             },
-            'Waze': coord => {
-                const link = 'https://www.waze.com/ul?ll={0}%2C{1}&navigate=yes';
-                return link.format(coord.lat, coord.lng);
+            'Google Multi': {
+                'getLink': (dst, wpts) => {
+                    // Undocumented URI for multi-waypoint route planning.

```

Comment:
I dont have name translation for this yet, and I could not think of a better name. It would require a description probably, but there is not interface for that right now.

Reply:
Google through the selected route/path? I mean that is what it does essentially